backend/src/categories/words.ts

```diff
@@ -7,23 +7,27 @@ import {
   PrimaryGeneratedColumn,
 } from 'typeorm';
 import { Category } from './category';
+import { Choices } from './choices';
 
 @Entity('words')
 export class Words {
   @PrimaryGeneratedColumn()
   word_id: number;
 
-  @Column({ nullable: false })
+  @Column({ default: 1 })
```

Comment:
I think we can revert this back to `nullable: false` since I don't think it should default to 1